client_packages/chat/index.html

```diff
@@ -0,0 +1,15 @@
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="style/checkbox.css" media="screen">
+        <link rel="stylesheet" type="text/css" href="style/main.css" media="screen">
+        <meta charset="utf-8" />
+    </head>
+    <body>
+        <div id="chat" class="ui_element">
```

Comment:
Can you please use more meaningful class name here? Something like ‘chat_wrapper’